Specs/NumberWithUnit/Italian/DimensionModel.json

```diff
@@ -624,5 +624,15 @@
         }
       }
     ]
+  },
+  {
+    "Input": "l'amministrazione",
```

Comment:
Can you add cases with real usage of sentences in Italian with something like "10l" and "20 l" to show that litter is properly extracted in those?
Or see if any specs currently contain litter examples but are marked as NotSupported in .NET?